spring-cloud-dataflow-aggregate-task/src/main/java/org/springframework/cloud/dataflow/aggregate/task/AggregateTaskExplorer.java

```diff
@@ -97,7 +98,17 @@ public interface AggregateTaskExplorer {
 	 * @param completed Indicator to find only completed tasks
 	 * @return list of task executions
 	 */
-	List<AggregateTaskExecution> findTaskExecutionsByName(String taskName, boolean completed);
+	List<AggregateTaskExecution> findTaskExecutions(String taskName, boolean completed);
+
+	/**
+	 * Get a list of executions for a task by name, completion status and end time.
+	 *
+	 * @param taskName  the name of the task to be searched
+	 * @param completed Indicator to find only completed tasks
```

Comment:
The Javadoc does explain the meaning, but I think naming `onlyCompleted` may be more consistent w/ the other newly introduced APIs and also more readable/clear.

I will only comment that once here but it applies to multiple locations in this PR.

Reply:
I will also change this to the method above as it is called the same there. 👍